react/README.md

```diff
@@ -279,6 +279,22 @@
   <div />
   ```
 
+- Avoid using array index as `key` prop. ([why?](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318))
```

Comment:
Use unique element ids and not array indexes or other counter indexes as React may display elements out of order. See this article for why.